packages/backend/src/graphql/resolvers/outgoing_payment.test.ts

```diff
@@ -539,6 +539,103 @@ describe('OutgoingPayment Resolvers', (): void => {
     })
   })
 
+  describe('Mutation.cancelOutgoingPayment', (): void => {
+    let payment: OutgoingPaymentModel
+
+    beforeEach(async (): Promise<void> => {
+      const { id: walletAddressId } = await createWalletAddress(deps, {
+        assetId: asset.id
+      })
+      payment = await createPayment({ walletAddressId })
+    })
+
+    const states: [string, OutgoingPaymentError | null][] = Object.values(
+      OutgoingPaymentState
+    ).flatMap((state) => [
+      [
+        'Not enough balance',
+        state == OutgoingPaymentState.Funding
+          ? null
+          : OutgoingPaymentError.WrongState
+      ],
+      [
+        'Missing KYC',
+        state == OutgoingPaymentState.Funding
+          ? null
+          : OutgoingPaymentError.WrongState
+      ]
+    ])
+    test.each(states)(
```

Comment:
I think we should do this kind of testing in the outgoing payment service tests instead (since that's where the logic for checking the status lives), while these resolver tests can just worry about a happy path and a single error path where the correct error result gets returned